### ISSUE TITLE
Documentation fixes

### ISSUE THREAD

**Issue by valacar** [facelessuser/RegReplace#30]
*Back References* section:
- [x] wrong formatting of `\l` ...you used single quotes instead of the backtick character.
- [x] "when performing certain kinds or replaces", should probably be "when performing certain kinds **of** replaces".
- [x] you also say you can enable extended back references by setting `"extended_back_references": false` ...should be set to true to enable it.

*Use Regex on Entire File Buffer when Using Selections* section:
- [x] "opposed the default behavior" should probably be "**as** opposed **to** the default behavior"

Also, I just want to say thanks for documenting the extended back references.  A few days ago I was try to uppercase HTML/XML tags, and now I know why `\U` didn't work.  I'm guessing you had to replace it with `\C` and `\c` because of Python's unicode escape character.

**Comment by facelessuser** [comment]
Thanks for reporting the doc stuff.  I often miss stuff like that.  The brain sees what I am thinking, not always what I am writing :smile:.

> Also, I just want to say thanks for documenting the extended back references. A few days ago I was try to uppercase HTML/XML tags, and now I know why \U didn't work. I'm guessing you had to replace it with \C and \c because of Python's unicode escape character.

No problem. It is an extremely useful feature.  I wish I could have used `\u` and such, but sadly, I couldn't due to the unicode escapes.  I figured `\c` for **capitalize** was an okay alternative, but yeah, unless it's documented, no one knows.  Figured I should finally get around to writing it down.
